spec/view_objects/providers/allocations_view_spec.rb

```diff
@@ -0,0 +1,61 @@
+require "rails_helper"
+
+describe AllocationsView do
```

Comment:
I'm assuming that because the allocation statuses are unit tested we do not need to also test in the allocation feature spec